fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -257,6 +257,8 @@
     public static final Predicate<Property> isManagedPredicate =
         hasFedoraNamespace.or(p -> managedProperties.contains(p));
 
+    public static final String NT_VERSION_FILE = "nt:versionFile";
```

Comment:
Please add a comment to this constant indicating that it is actually a Fedora-defined type for the purpose of...